charts/postgres-operator/crds/postgresqls.yaml

```diff
@@ -173,6 +173,10 @@ spec:
               type: object
               additionalProperties:
                 type: string
+            serviceAnnotations:
```

Comment:
I've applied alphabetical sorting here to make it easier to find parameters because we have so many now. Can you move it down, please?